### PR TITLE
Patch 2019-10-16 Halloween new weapon and chara

### PR DESCRIPTION
**Pull request by kei-gbf**
This is rough patch, I expect brush up process in the review.

## New Characters
- [x] 真紅と冥闇 ゼタ＆バザラガ(ハロウィン) [Zeta and Vaseraga (Halloween)](https://gbf.wiki/Zeta_and_Vaseraga_(Halloween))
  - [x] DA: 1000% and TA
- [x] ハレゼナ(ハロウィン) [Hallessena (Halloween)](https://gbf.wiki/Hallessena_(Halloween))
  - [x] TA: 1000% and ougigage down 35%
  - [x] Skip Stackable support ATK/DEF debuff
- [x] ザザ(ハロウィン) [Zaja (Halloween)](https://gbf.wiki/Zaja_(Halloween))

## New Weapon

- [x] SSR [Enchanted_Broomstick](https://gbf.wiki/Enchanted_Broomstick)
- [x] SSR [Spirit Seeker](https://gbf.wiki/Spirit_Seeker)
  - [x] TODO: uncap Spirit Seeker 2nd skill
    global_logic.js one_night_party has temporary amount for test.
- [x] Skip SR [Horrible Voulge](https://gbf.wiki/Horrible_Voulge)

Test Data for import (JSON) [2019-10-16.json.txt](https://github.com/MotocalDevelopers/motocal/files/3838957/2019-10-16.json.txt)

## HELP WANTED

- amount of the skill "one night party" / 第２スキルの効果量の検証情報

## Known issue

- translation error 不詳
  master repo has this, not fix in this PR
- Confuse point in translation, ウィッチブルーム direct translation is "Witch's Broom"
  but English locale already has the name, the new weapon is Enchanted Broom in English.

### CODE REVIEW

**Comment by Groovman** [comment]
With Spirit Seeker you can assume 2nd is small normal atk + small trium just with axe as condition, doesn't look like something special, just a unique name.

File: armData.json [hunk 1]
```diff
@@ -13351,6 +13351,46 @@
...
+        "type": "axe",
+        "skill1": "normalSanteL",
+        "skill2": "one_night_party",
+        "element2": "light",
+        "skill3": "non",
+        "element3": "light",
+        "minhp": 20,
+        "minattack": 500,
+        "hp": 120,
+        "attack": 2986,
```
Comment (kagekiyo7) [inline armData.json]:
```js
        "hp": 120,
        "attack": 2986,
```
```js
        "hplv100": 150,
        "attacklv100": 3610,
```

hp, attack and hplv100, attacklv100 values are opposite.
([4凸]スピリットシーカー)

File: charaData.json [hunk 2]
```diff
@@ -659,6 +659,50 @@
         "imageURL": "./charaimgs/3040169000_01.png",
         "en": "Nier"
     },
+    "真紅と冥闇 ゼタ＆バザラガ(ハロウィンver)": {
+        "name": "真紅と冥闇 ゼタ＆バザラガ(ハロウィンver)",
+        "ja": "真紅と冥闇 ゼタ＆バザラガ(ハロウィンver)",
+        "element": "earth",
+        "type": "balance",
+        "race": "human",
```
Comment (kagekiyo7) [inline charaData.json]: Correct race are fuman and doruf.
(真紅と冥闇 ゼタ＆バザラガ(ハロウィンver))
image from internet
image

File: charaData.json [hunk 3]
```diff
@@ -659,6 +659,50 @@
...
+    "ハレゼナ(ハロウィンver)": {
+        "name": "ハレゼナ(ハロウィンver)",
+        "ja": "ハレゼナ(ハロウィンver)",
+        "element": "light",
+        "type": "attack",
+        "race": "doraf",
+        "sex": "female",
+        "fav1": "axe",
+        "fav2": "none",
+        "support": "none",
```
Comment (kagekiyo7) [inline charaData.json]: Support is ougi buff 35%DOWN. (in the future)
https://xn--bck3aza1a2if6kra4ee0hf.gamewith.jp/article/show/172848

Comment (kagekiyo7) [reply charaData.json]: and 
・攻撃力10%UP×ベリナイLv(別枠乗算/最大50%)
・防御力6%DOWN×ベリナイLv(最大30%)

**Pull request merged**